packages/web5/lib/src/crypto/in_memory_key_manager.dart

```diff
@@ -20,7 +20,7 @@ import 'package:web5/src/crypto/crypto.dart';
 /// final signatureBytes = await keyManager.sign(keyAlias, Uint8List.fromList([20, 32]));
 /// ```
 ///
-class InMemoryKeyManager implements KeyManager, KeyImporter, KeyExporter {
+class InMemoryKeyManager implements KeyManager {
```

Comment:
the reason i split this out was to make it easier to [accommodate](https://github.com/TBD54566975/web5-dart/blob/main/packages/web5/lib/src/dids/bearer_did.dart#L36-L38) concrete `KeyManager` implementations that don't support exporting. e.g. a key manager that leverages [iOS Secure Enclave](https://support.apple.com/guide/security/secure-enclave-sec59b0b31ff/web) doesn't have the ability to export keys. 

Technically an `IosEnclaveKeyManager` could just throw an `UnsupportedOperationException` from `exportKey` but it's harder to require that. 

we've also got these same interfaces across languages e.g. [kotlin](https://github.com/TBD54566975/web5-kt/blob/c1c8f3ee828577b28cfc3c972b4b0ec3e552bae8/crypto/src/main/kotlin/web5/sdk/crypto/KeyManager.kt#L64-L78) [go](https://github.com/TBD54566975/web5-go/blob/main/crypto/keymanager.go#L23-L26)